Ascend-Server.api/Program.cs

```diff
@@ -28,18 +63,21 @@
            .AllowAnyMethod();
 });
 
-
-if (app.Environment.IsDevelopment())
-{
-    app.UseSwagger();
-
-    app.UseSwaggerUI();
-}
+app.UseSwagger();
```

Comment:
This is a duplicate call. Remove this one because it's too high up in the pipeline anyway.